Tests/test_font_bdf.py

```diff
@@ -8,13 +10,20 @@
 
 
 def test_sanity() -> None:
-    with open(filename, "rb") as test_file:
-        font = BdfFontFile.BdfFontFile(test_file)
+    with open(filename, "rb") as fp:
+        font = BdfFontFile.BdfFontFile(fp)
 
     assert isinstance(font, FontFile.FontFile)
     assert len([_f for _f in font.glyph if _f]) == 190
 
 
+def test_valueerror() -> None:
```

Comment:
More descriptive name, as the test doesn't directly deal with the `ValueError`:
```suggestion
def test_zero_width_chars() -> None:
```